### PR TITLE
Fix w3cid and added document status

### PR DESCRIPTION
**Pull request by eladalon1983**
Also fixes a typo - s/w3cId/w3cid

***

### :boom: Error: 500 Internal Server Error :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Apr 4, 2022, 10:12 AM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Fraw.githubusercontent.com%2Feladalon1983%2Fmediacapture-region%2F81c0c65b88205a976423d89c06f0f5413a87c20a%2Findex.html%3FisPreview%3Dtrue)

```
waiting for function failed: timeout 30000ms exceeded
```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c/mediacapture-region%2338.)._
</details>

### CODE REVIEW

File: region-capture.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 var respecConfig = {
   group: "webrtc",
-  specStatus: "ED",
+  specStatus: "FPWD",
```
Comment (dontcallmedom) [inline region-capture.js:3]:
```suggestion
  specStatus: "ED",
```
The version in the repo should remain as "ED" - it is when it gets published under /TR/mediapture-region that the proper status will be set through the publication system.

**Comment by eladalon1983** [comment]
@dontcallmedom, I see the following line in Bernard's [latest email](https://lists.w3.org/Archives/Public/public-webrtc/2022Apr/0002.html):
> With the blocking comments resolved, we declare WG consensus to proceed with FPWD publication of "Region Capture".

Could you take the necessary actions to move this document to FPWD, then?

**Comment by dontcallmedom** [comment]
yes, it's already in progress and on path to be published tomorrow as a FPWD

**Comment by eladalon1983** [comment]
Great. Shall I close this PR, then?

**Comment by dontcallmedom** [comment]
merging it would be good :)

**Pull request merged**